SimplenoteScreenshots/SimplenoteScreenshots.swift

```diff
@@ -303,17 +304,3 @@ extension XCUIElement {
         }
     }
 }
-
-extension XCUIApplication {
-
-    func isDeviceIPhone8Plus(_ device: XCUIDevice = .shared) -> Bool {
-        let iPhone8PlusScreenHeight = CGFloat(736)
-
-        let frame = windows.element(boundBy: 0).frame
-
-        switch device.orientation {
-        case .landscapeLeft, .landscapeRight: return frame.width == iPhone8PlusScreenHeight
-        case _: return frame.height == iPhone8PlusScreenHeight
-        }
-    }
-}
```

Comment:
This logic is now part of the framework. The only changed needed in this file was to `import` it.